src/components/forms/badge-form.js

```diff
@@ -114,11 +126,39 @@ class BadgeForm extends React.Component {
                 </div>
                 <div className="row form-group">
                     <div className="col-md-4">
-                        <label> {T.translate("edit_ticket.print_count")}:&nbsp;</label>
-                        {entity.printed_times}
+                        <label> {T.translate("edit_ticket.print_excerpt")}:&nbsp;</label>
+                        <table className="table table-striped table-bordered">
+                            <thead>
+                                <tr>
+                                    <th>{T.translate("edit_ticket.type")}</th>
+                                    <th>{T.translate("edit_ticket.count")}</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {Object.keys(entity.print_excerpt).map((row, i) => {
+                                    let rowClass = i % 2 === 0 ? 'even' : 'odd';
+                                    return (
+                                        <tr id={row} key={'row_' + row} role="row" className={rowClass}>
+                                            <td>{row}</td>
+                                            <td>{entity.print_excerpt[row]}</td>
+                                        </tr>
+                                    );
+                                })}
+                            </tbody>
+                        </table>
+                    </div>
+                    <div className='col-md-4'>
```

Comment:
only show print ui ( excerpt , ddl and button ) if ticket badge has IN_PERSON access level